### PR TITLE
Update usecases.json

### PR DESCRIPTION
**Pull request by clupascu**
Added video tutorial for "Build your own single hippocampal cell model using HBP data" usecase

### CODE REVIEW

File: src/assets/config_files/usecases.json [hunk 1]
```diff
@@ -626,6 +626,7 @@
           "src": "https://raw.githubusercontent.com/antonelepfl/usecases/dev/src/assets/images/objectives.png"
         },
         "next": "sc_form",
+	"tutorial": "https://object.cscs.ch/v1/AUTH_c0a333ecf7c045809321ce9d9ecdfdea/web-resources-bsp/bsp-video-tutorials/BuildYourOwnSingleHippocampalCellModel/index.html",
```
Comment (antonelepfl) [inline src/assets/config_files/usecases.json]: Hi Carmen, Could you please indent this new element the same way that you have the other so it's more readable ?

**Comment by clupascu** [comment]
I see the line correctly indented. Could you check again please?

**Comment by antonelepfl** [comment]
Locally 
Screenshot 2019-12-10 at 12 08 48

Github
Screenshot 2019-12-10 at 12 09 11

**Comment by antonelepfl** [comment]
You can check the files that were modified going to the [Files Changed](https://github.com/antonelepfl/usecases/pull/273/files) tab

**Comment by clupascu** [comment]
When I click on View File or  Edit File though I see it correctly indented

Schermata 2019-12-10 alle 12 16 32

I can't understand what is happening.

**Comment by alex4200** [comment]
Maybe it is an issue with a tab?

**Comment by antonelepfl** [comment]
Yeah it is an issue with the tab. Please convert the tabs into spaces

**Comment by clupascu** [comment]
Done. Can you please check now? Thanks.

**Pull request merged**